src/daemon/rpmostreed-daemon.c

```diff
@@ -422,8 +422,12 @@ rpmostreed_daemon_reload_config (RpmostreedDaemon *self,
         return FALSE;
     }
 
-  self->ex_stage_deployments = g_key_file_get_boolean (config, EXPERIMENTAL_CONFIG_GROUP,
-                                                       "StageDeployments", NULL);
+  /* The default used to be `false`, so check for the key being present */
+  if (g_key_file_has_key (config, EXPERIMENTAL_CONFIG_GROUP, "StageDeployments", NULL))
```

Comment:
Maybe just one release with that, and then we drop support for the key entirely? Or were you thinking of keeping around support for `StageDeployments=false`? I'm leaning more towards just not supporting turning it off.

Reply:
Yeah, agree.

Reply:
In this PR there's a build-time option; @dustymabe was arguing that this should go into rawhide first or something.  I dunno if that's really worth it.  We have FAHC for automated testing before release.

Reply:
No strong opinions either way. Seems relatively low risk at this point, but I'm fine with delaying f28 for one more release or so. (Maybe at the same time we drop support for the `StageDeployments` option completely),